pkg/collector/corechecks/cluster/common.go

```diff
@@ -32,3 +34,13 @@ func RunLeaderElection() (string, error) {
 
 	return leaderEngine.GetLeader(), nil
 }
+
+// SetCacheStats sets the cache stats for each resource
+func SetCacheStats(resourceList interface{}, resourceMsgs interface{}, nodeType orchestrator.NodeType) {
```

Comment:
We can use the lengths directly, instead of inputting an interface here

Reply:
Thanks, Just updated it